.gitignore

```diff
@@ -94,4 +94,9 @@ iOSInjectionProject/
 .idea/
 secrets.xcconfig
 
-build_for_unit_testing/
\ No newline at end of file
+#Unit tests
+build_for_unit_testing/
+
+#Build
+build_output/
+Package.resolved
```

Comment:
Not sure if we want this in here, its in the app project so that the app will automatically update the versions when building. That's not a thing I think we want this repo doing?